package.json

```diff
@@ -26,6 +26,7 @@
     "test": "yarn workspace @nufi/cardano-metamask-snap run test"
   },
   "resolutions": {
+    "@cardano-ogmios/client/ws": "7.5.10",
```

Comment:
I believe this resolution is redundant, just remove it, try `yarn install` and if yarn.lock doesn't change (which I expect) it should be ok to leave the resolution out, which is unnecessarily fragile